app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
-    #first = last, because of ordering in model
-    weight = self.weight
-    height = self.person.length
+    height = person.length
```

Comment:
You are totally right. I use height here, because in my initial dbase setup, I accidentally chose the wrong term. I will change it in my dbase.